src/ui/public/notify/partials/toaster.html

```diff
@@ -7,7 +7,9 @@
 
         <i class="fa" ng-class="'fa-' + notif.icon" tooltip="{{notif.title}}"></i>
 
-        <kbn-truncated source="{{notif.content | markdown}}" is-html="true" length="{{notif.truncationLength}}" class="toast-message" /></kbn-truncated>
+        <kbn-truncated ng-if="notif.content" source="{{notif.content | markdown}}" is-html="true" length="{{notif.truncationLength}}" class="toast-message" /></kbn-truncated>
```

Comment:
Can we make this multiline? This is pretty hard for me to read, sort of like reading a function call with a lot of inline arguments. I think multiline will also make future diff comparisons easier.

```html
<kbn-truncated
  ng-if="notif.content"
  source="{{notif.content | markdown}}"
  is-html="true"
  length="{{notif.truncationLength}}"
  class="toast-message"
/></kbn-truncated>
```